components/Input/Input.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import { Meta, StoryFn } from "@storybook/react";
+import Input from "./Input";
+
+export default {
+  title: "Input",
+  component: Input,
+} as Meta<typeof Input>;
```

Comment:
Adicionalmente, podemos pasarle los `args` aca, asi se agregaran "por defecto" a todas las historias (siempre que estas las reciban).

En este caso agregar:
```suggestion
export default {
  title: "Input",
  component: Input,
  args: {
    required: false,
  },
} as Meta<typeof Input>;
```

hara que todas las historias tengan esa prop como `false`, excepto la que cambia a `true`